functionality/database/src/main/kotlin/pl/patrykgoworowski/mintlift/functionality/database/converter/JsonConverters.kt

```diff
@@ -0,0 +1,28 @@
+package pl.patrykgoworowski.mintlift.functionality.database.converter
```

Comment:
```suggestion
package com.patrykandpatryk.liftapp.functionality.database.converter
```

I’d also suggest renaming the packages accordingly.